pennylane/operation.py

```diff
@@ -508,7 +508,7 @@ class Operation(Operator):
     * :attr:`~.Operation.generator`
 
     Args:
-        params (tuple[float, int, array, Variable]): operation parameters
+        data (tuple[float, int, array, Variable]): operation parameters
```

Comment:
This one should remain `params`, as that is what the `__init__` method below uses for the argument name :)

Reply:
```suggestion
        params (tuple[float, int, array, Variable]): operation parameters
```